pkgs/tools/security/cve-bin-tool/default.nix

```diff
@@ -69,18 +68,17 @@ buildPythonApplication rec {
     pytestCheckHook
   ];
 
+  # Wants to open a sqlite database, access the internet, etc
```

Comment:
A sqlite database can be created with touch.

Reply:
Looks like that `cve-bin-tool` need a do over anyways.